### DISCUSSION TITLE
I want to contribute to AppointmentStore Documentation.

### DISCUSSION THREAD

**Discussion by devarsh10** [unoplatform/uno#13967] [Q&A] [answered]
I want to contribute to the issue #10384. However, I am unable to move forward and not able to understand the further process. Can someone please guide me, with the further steps?

Regards,
Devarsh Shah.

**Comment by limeniye** [comment]
Hello, @devarsh10 !
Thank you very much for your interest in Uno contributing. 

I saw something at 17:00 in this stream by @MartinZikmund

[Contributing to Uno Platform 101](https://www.youtube.com/watch?v=1LnYZQVZFfs&list=PLC0wF4Qw6SDNu61eMpdGCw-c5RR2Tk2R6&index=35&t=1020s&ab_channel=MartinZikmund)

I hope it will helps you :3

**Reply by MartinZikmund** [reply, marked as answer]
@devarsh10 As @limeniye mentioned, it would be great if you check the linked video for some basic information on how to contribute, it contains the required details for documentation PRs as well. If you have further questions, please post here, happy to help!

**Reply by devarsh10** [reply]
Hey @MartinZikmund, I have been looking for AppointmentStore API but I am unable to understand it's use-case. So, can we connect so that I can understand it and write the documentation for the same?
Sorry for the delayed response.
